### ISSUE TITLE
1.4.29V0-V2 - Feedback from xmage.de 

### ISSUE THREAD

**Issue by LevelX2** [magefree/mage#4864]
# :heavy_check_mark:  Feedback from 1.4.29V0
- [x] Sergeant-at-Arms is buged: you get the tokens without kicking it (reported by juliancat21 / 2018-04-19 22:16:34.291).
- [x] Benalish Honor Guard: Benalish Honor Guard is supposed to only count legendary creatures. It currently also counts other legendary permanents (tested on uncrewed Weatherlight). (reported by Bob49 / 2018-04-19 22:19:53.877).
- [x] Phyrexian Scriptures bug: It does not exile my own yard (reported by Azeraph / 2018-04-19 22:52:51.955).
- [x] Corrosive Ooze: Ooze puts the trigger on the stack correctly, but nothing happens (reported by oOoOoOo / 2018-04-19 22:58:47.23).
- [x] KNight of Malice bug: (300 characters max)KNight of malice seemed to give everything a +1/+0 (reported by phibes / 2018-04-19 22:59:38.903).
- [x] Wild Onslaught: It put 2 counters on all creatures even though it wasn't kicked. (reported by MadMoses / 2018-04-19 22:59:54.826).
- [x] match crash: undoing or cancelling a spell crashes game and gives oponnent a win (reported by phibes / 2018-04-19 23:01:22.851). **JayDi85: Can't reproduce**
- [x] Knight of Malice: Knight of malice is functioning as an anthem, teamwide buff rather than just himself. (reported by DrLaughter / 2018-04-19 23:01:50.769).
- [x] Vicious Offering: Doesnt select a target on cast, was tried when kicked. (reported by DrLaughter / 2018-04-19 23:23:40.273).
- [x] Rampaging Cyclops: didn't get -2/-0 although blocked by a creature and a saproling token (reported by MadMoses / 2018-04-19 23:39:20.542).
- [x] Knight of Grace: Gave _all_ my creatures +1/+0 instead of only itself. (reported by MadMoses / 2018-04-19 23:45:05.552).
- [x] vicious offering: doesn't work (reported by Lathaev / 2018-04-19 23:54:05.735).
- [x] Seismic Shift DOM: Not allowing me to select two creatures that my opponents c (reported by McNasty / 2018-04-19 23:54:06.973). **LevelX: Can't reproduce**.
- [x] Knight of Grace: Whenever my oponent plays a black permant, Knight of Grace gives all of my creatures +1/ +0, until end of turn. It should only be Knight of grace that gets the boon (reported by McNasty / 2018-04-19 23:55:09.384).
- [x] Knight of grace bug: Knight of Grace gives all creature owner controls +1/+0 for a black permanant being on the battlefield rather than just knight of Grace (reported by No1OConsequens / 2018-04-19 23:55:13.74).

# :x: Feedback from 1.4.29V1
- [ ] Rollback: It seems that whenthe opponent asks for a Rollback, my turn just skips automatically. My opponent has to rollback again in order to go back to normal. (reported by The_Vigil / 2018-04-21 13:28:15.171).
- [ ] view hideaway lands: it would be nice to see what I put under a hideaway land in case it slips my mind (reported by Knighthawke / 2018-04-21 23:13:34.072).
- [ ] Commander game priority freeze: Whilst playing commander priority seemed to be given to two players at the same time which caused the game to freeze (reported by JoElVago / 2018-04-22 04:24:57.592).
- [ ] soubond bug: soul bond is not triggering when another creature enters the battlefield (reported by Knighthawke / 2018-04-22 05:34:44.993).
- [ ] saga's aren't working as expected: According to the rules, sagas should be on battlefield until after the third chapter resolves. Right now I can't bounce the saga in response to the third chapter because they leave the
- [ ] xmage still doesnt work properly with multiplayer games and conceding: xmage still doesnt work properly with multiplayer games and conceding (reported by EvilGeek / 2018-04-20 18:10:02.031).
- [x] knight of malice: i control a knight of malice and pegasus courser, knight is not getting +1/+1 (reported by h1jack / 2018-04-20 02:13:28.911).
- [x] Knight of Malice is bugged: Should get +1/+0 if the controller has a white permanent. (reported by TMBR / 2018-04-20 02:19:44.256).
- [x] knight of malice: doesnt't get +1/+1 when i control a white permanent (reported by LordMalphas / 2018-04-20 02:51:06.492).
- [x] corrosive ooze bug: when blocking an equipped creature, the equipment was not destroyed. (reported by n1ght1ng4le / 2018-04-20 02:53:39.32).
- [x] knight of grace: doesn't get +1 when someone controls a black permanent, it should (reported by LordMalphas / 2018-04-20 03:00:48.869).
- [x] naban and caligo skin witch: believe ruling is permanents are on the battlefield when etb triggers therefore skin witch should trigger twice unless there is a new ruling (reported by Will / 2018-04-20 03:29:40.377).
- [x] Knight of Grace doesn't pump: This doesn't pump when a black permanent is in play. (reported by jerjern / 2018-04-20 04:14:45.845).
- [x] your shit is still fucking junk: (300 characters max)wtf. stopped playing on here because of your shuffler 6 months ago and not better fucking junk (reported by osamamyass / 2018-04-20 04:26:44.66).
- [x] junk shit: fucking junk (reported by osamamyass / 2018-04-20 04:39:26.887).
- [x] junk shity: wtf such fucking bullshit (reported by osamamyass / 2018-04-20 04:41:55.134).
- [x] two headed giant: when attacking with two headed giant, both coins came up tails (should give menace), but opponent was able to block with one creature (reported by h1jack / 2018-04-20 05:21:53.727).
- [x] knight of malice: Played knight of malice, then radiant destiny naming knights. knight of malice became a 3/3 from radiant destiny, but did not get the +1/+0 from a white permanent being on the battlefield. (reported by Haywire / 2018-04-20 05:49:06.886).
- [x] knight of grace: doesn't get +1/+0 when my opponent controls slimefoot, the stoway. knight entered first (reported by Haywire / 2018-04-20 06:05:00.927).
- [x] Knight of Malice: should get +1/+0 for opponent having white permanent thank you for your hard work (reported by Will / 2018-04-20 06:22:46.826).
- [x] Treasure tokens and antiquities war: All my treasure tokens went to graveyard after the end of the turn in which antiquitiy war's third ability resolved. Also, artifacts that enter after antiquities war's third ability resolves enter as artifact creatures when they shouldn't (reported by Haywire / 2018-04-20 06:48:07.155).
- [x] Board the Weatherlight [DOM] Bottom of Library stack: Board the weatherlight says that the cards are placed on the bottom of the library in a random order, but the game lets you stack the order. (reported by codecimal / 2018-04-20 06:57:47.236).
- [x] Rampaging Cyclops bug: Rampaging Cyclops did not get -2/-0 when blocked by two creatures. (reported by Meowmix / 2018-04-20 09:17:16.819).
- [x] Knight of Grace bug: Knight of Grace did not get +1/+0 when an opponent controlled a black creature. (reported by Meowmix / 2018-04-20 09:19:00.619).
- [x] same fucking: (300 characters max)why did i ever came back to play on here? you shit is fucking junk ya some fucking really fucking random fucking junk (reported by osamamyass / 2018-04-20 14:12:37.444).
- [x] Rat Colony rules text: Rat Colony's rules text is wrong. It says Rat Colony gets +1/+0 for each other creature you control. It should sy "for each other Rat you control". (reported by bhegstad / 2018-04-20 15:11:39.93).
- [x] Rampaging Cyclops: It did not get -2 when I blocked with2 creatures (reported by stygi4n / 2018-04-20 15:43:40.672).
- [x] Rites of Bezlenok/general Saga issue: Rites of Bezlenok lore counters trigger as an ETB allowing removal to prevent the first chapter from occuring. Counters aren't suppsed to be a trigger, first chapter should have resolved through removal. Might be a general Saga issue. (reported by EphMynor / 2018-04-20 15:45:34.087).
- [x] caligo skin witch: kicker ability seems to act as cast ability rather than etb (reported by Lathaev / 2018-04-20 16:00:40.733).
- [x] Sagas: Sagas auto sac themselves as soon as 3 triggers. Shouldnt sac until ability resolves allowing time to do something to the saga such cast rescue (reported by Lathaev / 2018-04-20 16:16:58.057).
- [x] fuck you: (300 characters max)give me a goddamn fucking break every fucking i must mulligan and every fucking can t get lands ya real fucking random (reported by osamamyass / 2018-04-20 17:15:15.423).
- [x] such junk: wtf really every fucking game (reported by osamamyass / 2018-04-20 17:31:46.825).
- [x] Knight of Grace does not get +1/+0 from opponent controlling a black permanent: (300 characters max) (reported by Carl / 2018-04-20 17:47:04.003).
- [x] are you fucking kidding: fucking bullshit total fucking bullshit no wonder i stopped playing on here such fuckingh junk (reported by osamamyass / 2018-04-20 17:47:23.71).
- [x] Knight of Malice/Grace only get +1/+0 if BOTH players control a permanent of that color: Probably an "and" that should be an "or" (reported by Carl / 2018-04-20 17:48:26.48).
- [x] unfucking beleivable: dude this has got to be the worst fucking shuffler ever who did it for you some fucking moron? (reported by osamamyass / 2018-04-20 18:38:49.472).
- [x] dude this is fucking retarded fix your fucking algarithim with your shuffler: such fucking junk (reported by osamamyass / 2018-04-20 18:47:46.356).
- [x] Knight of Grace: should get +1+0 when opponent controls a black permanent thank you for your hard work on this project (reported by Will / 2018-04-20 19:15:07.945).
- [x] knight of malice bugged: doesn't recieve +1/+0 bonus if you control a white permanent (reported by 1 / 2018-04-20 19:29:58.651).
- [x] Dominaria - black/white hexproof knights: They don't get the +1/+0 bonus (reported by Soste / 2018-04-20 19:43:41.993).
- [x] Muldotha is bugged: Aparently it was wrongly coded and it can cast or play stuff from an enemy graveyard (reported by JohnLemon / 2018-04-20 20:07:55.154).
- [x] phyrexian scriptures: third lore trigger exiles all graveyards, not just opponent's (reported by h1jack / 2018-04-20 20:43:21.052).
- [x] Knight of Grace: Knight of Grace dont get the +1/+0 when a player have a black permanent (reported by avatara / 2018-04-20 21:46:20.821).
- [x] DOM- knight of grace not triggering 3/2 when black perm: (300 characters max) (reported by ProBonoGigolo / 2018-04-20 22:00:32.199).
- [x] DOM Knight Of Malice not working: Hi,just wanted to inform you that Knight of Malice is not working properly, he doesn't get the +1/+0 from white permanents. (reported by Lynch785 / 2018-04-20 22:46:08.506).
- [x] knight of grace: knight of grace did not get buffed with black perm in play (reported by n1ght1ng4le / 2018-04-21 01:28:43.45).
- [x] Jaya balar's emblem is glitched: using her ult is supposed to make you able to cast spells from the graveyard but as of right now tryng to cast cut//ribbon from the graveyard forced xmage into an error were it automaticly rolledback to before jaya ulted (reported by GD1112 / 2018-04-21 01:51:22.417).
- [x] Knight of Grace / Knight of Malice: Hi, the hability of these cards is not triggering when another permanent of the mentioned color is in play. (reported by Apprentice / 2018-04-21 02:13:24.293).
- [x] knight of grace: Not getting +1/+0 when your opponent controls a black perment (reported by McNasty / 2018-04-21 02:27:06.484).
- [x] Bug Report: Couldn't cast Primevals' Glorious Rebirth that was exiled with Rona while Rona was still on the field. (reported by Arimus / 2018-04-21 03:39:40.776).
- [x] rona is broken: when i cast rona the first time her ability works fine but then if i blink her it doesnt work, as in i cant cast the exiled card. (reported by igniteaxiom / 2018-04-21 04:38:16.205).
- [x] Knight of the grace bugged: =+1/+0 doesnt work on Knight ond the grace, and maybe in the black knight too (reported by JimRed / 2018-04-21 13:11:46.264).
- [x] bug: the card "Ancient animus" always fights, not only if the target is a legendary the card "Juggernaut" doesn't need to attack, also when it is able to attack both cards are from dominaria keep up the good work (reported by casecold / 2018-04-21 13:30:13.682).
- [x] Knight of Grace: Knight of Grace does not get +1/+0 when the player who controls Knight of Grace also controls a black permanent. (reported by Meowmix / 2018-04-22 02:11:35.168).
- [x] the antiquities war: after resolving the third chapter trigger, artifacts that enter the battlefield are 5/5 creatures (reported by h1jack / 2018-04-22 05:20:46.703).
- [x] Knight of Malice bug: Knight of Malice does not get +1/+0 when the player who controls Knight of Malice also controls a white permanent. It does get +1/+0 when an opponent controls a white permanent. (reported by Meowmix / 2018-04-22 05:26:45.541).
- [x] knight of grace: still does not get +1 for a player controloing a black permenant (reported by McNasty / 2018-04-22 05:29:51.172).
- [x] Inalla + Flash interaction: Inalla eminence ability isnt triggering when a creature enters the battlefield using the instant card Flash. (reported by Trinket / 2018-04-22 05:57:36.36).
 battlefield. (reported by wordra / 2018-04-22 06:49:30.492).
- [x] rona, disciple of gix: still broken. when i blink her she comes into play and her ability triggers but i can t cast the stuff that is exiled (reported by igniteaxiom / 2018-04-22 07:07:46.819).

# :x: Feedback from 1.4.29V2
- [ ] fix bugs: my opponent tapped eldrazi temple for 2 mana while my damping sphere was in play. this is ANOTHER bug as it doesnt work that way (reported by chaosz41 / 2018-04-22 13:15:01.875).
- [ ] Como rankean a los players: i dont understand, when i play a ranking match i lose 6 point, but when i win a ranking match i get a 3 point. (reported by whiteraven / 2018-04-23 04:30:12.773).
- [ ] dire fleet daredevil + wizard's lightning: i cast wizard's lightning from opponent's graveyard with dire fleet daredevil (no other creatures, no wizards on board) and it cost R, like with cost reduction applied (reported by Rybsson / 2018-04-23 19:11:07.715).
- [ ] Teshar bug: Teshar doesn't trigger on Sram (reported by casualsmash / 2018-04-24 01:23:48.443).
- [ ] timer issues: In my game, the opponent's clock was ticking and I had no interactions possible, but somehow I lost to idleness (reported by jerjern / 2018-04-24 04:03:55.792).
- [ ] eldrazy temple: Eldrazy temple still gives 2 mana under damping sphere to eldrazy spells (reported by Dadupe / 2018-04-24 11:53:41.708).
- [ ] Humility is bugged: I just finished playing a game where I had Humility on the board against firesong and sunspeaker where he played earthquake and still gained all the life from the earthquake. (reported by schulze961 / 2018-04-25 00:43:24.017).
- [ ] Pitiless plunderer and boardwipes: It specifically says in gatherer that if multiple things die at the same time as pitiless plunderer, I get those treasures. However, this doesn't happen currently (reported by Haywire / 2018-04-25 02:38:10.345).
- [ ] A scrollbar for your hand cards: When playing with infinite/ all Cards from your libary on your hand temporarely it would help to have a scrollbar to see all the cards (reported by Deathreaver / 2018-04-25 02:39:35.3).
- [ ] Master of the wild Hunt Bug: When Master of t.w.h activates his ability it asks the target of the ability to select a wolf, but you are not able to it may be because he has hexproof on wolves so you are not able to select them therefore youre unable to play the game (reported by HoodieChris / 2018-04-25 05:30:47.687).
- [ ] Asceticism + Master of the Wild Hunt = Broken Game: Using Asceticism with Master of the Wild Hunt's effect caused the game to want the opponent to choose creatures on mine to deal damage to but cannot due the hexproof effect that Asceticism gives. (reported by BigHam / 2018-04-25 05:35:52.177).
- [x] Sorting by color doesn't work.: When I hit the color tab in the Deck Editor nothing happens while it should sort the cards by their one color or multicolored. This has been broken for quite a few patches now. (reported by BigHam / 2018-04-25 05:37:54.896).
- [x] Cannot target Planewalkers: (300 characters max) (reported by Zapacheenie / 2018-04-22 08:23:40.76).
- [x] Any target: With Seismic Assault, I can't target Liliana planeswalkers (reported by The_Vigil / 2018-04-22 09:22:55.257).
- [x] Redirection: Walking Ballista couldn't target liliana of the veil, but damage could also not be redirected. (reported by AculAlHazred / 2018-04-22 11:18:08.054).
- [x] Walking ballista should be able to target planeswalkers: Just played a game and noticed that redirecting to planeswalkers was removed, but walking ballista ( potentially other cards) was not correctly errata'd to be able to target planeswalkers (reported by Immie / 2018-04-22 11:27:41.255).
- [x] Bug in Blightning: Hi, I played the card Blightning on an opponent. He lost 3 life, but doesn't have to discard two cards. Neither the opponent nor i had anything on the board which would have prevented the discarding. So i think there must be a Bug? Best regards Benjamin (reported by Belenus / 2018-04-22 16:35:53.268).
- [x] Dance of the dead bug: Dance of the dead caused my creature to enter the battlefield tapped. Its not supposed to do that. (reported by Overachiever / 2018-04-22 19:33:21.992).
- [x] damage wont redirect to planeswalkers: doesnt give an option (reported by BadMagicPlayer / 2018-04-22 21:13:13.175).
- [x] Sunschorched Desert: Hello, Sunschorched Desert can target creatures which the card normally can not target. (reported by Elec / 2018-04-23 01:20:16.229).
- [x] Bloodstone Goblin: While its ability triggers, the effect doesn't seem to happen despite it showing up in the game log. (reported by jerjern / 2018-04-23 02:56:04.597).
- [x] Rat Colony bug: I have multiple copies of Rat Colony in a commander deck and I was not allowed to use the deck even though the card states you can have any number of them (reported by ShaadiV / 2018-04-23 04:00:13.116).
- [x] Dominaria Rarity Error: Homarid Explorer from Dominaria is set as uncommon but should be uncommon (reported by Harzak / 2018-04-23 04:27:16.024).
- [x] Walking Ballista can't target planeswalker: When trying to target chandra torch of defiance walking ballista could not target her (reported by LandonQ / 2018-04-23 07:15:18.489).
- [x] typo in cabal paladin's mouseover text: text is missing a word or words (reported by grumpenprole / 2018-04-23 07:19:08.028).
- [x] My Battlesphere bug: Myr Battlesphere's attack ability triggers for every creature, not just the Battlesphere (reported by Adept78 / 2018-04-23 15:41:54.687).
- [x] Kolaghans Command cant redirect damage: fix it plz. thanks (reported by BadMagicPlayer / 2018-04-23 18:20:41.669).
- [x] Myr battlesphere bug: he does not do damage to the opponent by tapping myr and he triggers for all creature's instead of himself. the xmage text also says for each attacking creature instead of when myr battlesphere attacks (reported by Neko / 2018-04-23 21:20:19.553).
- [x] wtf.: (300 characters max)so your shuffler is junk but it the fuck off it 1/2 the time I serums vision if i only pick one card from scry to put to bottom this will put both to bottom (reported by osamamyass / 2018-04-23 22:32:55.098).
- [x] Can we have brawl please ? pretty please ?: Hi guys I'd very much like to see the addition of the new brawl format as a coded in format :D oh and you guys rock ^^ (reported by Aric_Haldan / 2018-04-24 01:00:22.795).
- [x] phyrexian scriptures: phyrexian scriptures on chapter 3 currently exiles all graveyards, instead of just opponents (reported by Sming / 2018-04-24 01:00:41.057).
- [x] fuk u: cant target jace (reported by kalabi / 2018-04-24 04:49:56.513).
- [x] Phyrexian Scritures Chapter 3: Scriptures chapter 3 shouldn't exile your graveyard (reported by GnarledOne / 2018-04-24 06:19:48.595).
- [x] Karn's Temporal Sundering Bugged: Return up to one target nonland forces exactly one nonland not up to. (reported by Electro / 2018-04-24 07:23:22.345).
- [x] Any target rules: It is not possible to select a planeswalker when you cast a damage spell with any target. And the damage can no longer be redirected. Thanks (reported by cnavax / 2018-04-24 13:45:31.57).
- [x] draws: reapier yourrrrrrrrrrr fuuuuuuuckinngggg drawsssss in this programmmm (reported by wilczek / 2018-04-24 14:12:35.996).
- [x] Just a Dominaria bug.: Rat Colony doesn't work in commander, the "any number of Rat Colony in your deck" bit. (reported by Eldritch_Error / 2018-04-24 14:41:56.558).
- [x] Searing Blaze bug: Searing Blaze does not work. It deals 6 damage to the creature instead of 3 damage to the creature and 3 to the player/planeswalker. (reported by Neopard / 2018-04-24 14:42:55.342).
- [x] Lava Spike Bug: Lava Spike is broken. Now it can target everything, even creatures. (reported by Neopard / 2018-04-24 14:44:14.637).
- [x] Planeswalkers cannot be targeted by burn spells: The rule change has not been implemented properly. (reported by leafwork / 2018-04-24 15:07:49.368).
- [x] Fix kolaghan´s command bug: When I cast kolaghan`s command and choose the 2 damage to any target mod, I cant do any damage to a planeswalker, If I click on the planeswalker, nothing happens and If I click on the player, he or she takes the damage directly. (reported by Todoacien / 2018-04-24 16:00:58.26).
- [x] kolaghan command: cant select chandra torch to deal 2 damage with K command (reported by Dadupe / 2018-04-24 16:50:28.488).
- [x] so much fun: (300 characters max)wow if this was real life playing the chances are you would see lands not only me but everyone play are either mana fucked flooded mulligan til nothing and conceding wow so much fun wtf. junk ass shit (reported by osamamyass / 2018-04-24 17:27:25.765).
- [x] junk: shit is junk (reported by osamamyass / 2018-04-24 18:26:01.194).
- [x] so fucking: wtf fucking junk as bullshit 1/2 the dont work and shuffler is the fucking worsted (reported by osamamyass / 2018-04-24 18:36:51.273).
- [x] wtf.: realy getting o;d with this shit no wonder I stopped playing om here (reported by osamamyass / 2018-04-24 19:09:32.103).
- [x] so fucking old: this is fucking total fucking junk bullshit (reported by osamamyass / 2018-04-24 19:20:35.794).
- [x] Cannot target planeswalker: Cannot target planeswalker with walking balista (reported by Iwanabtheguy / 2018-04-24 19:46:37.665).
- [x] sorcerer's wand bug: sorcerers wand hits creatures if equipped to vodalian arcanist. (reported by NaturalOrderer / 2018-04-24 20:53:59.704).
- [x] junk: junk junk (reported by osamamyass / 2018-04-24 22:04:30.224).
- [x] Skullcrack Broken: Skullcrack is not supposed to be able to target permanents, but XMage allows it to do so. (reported by iAreNubcake / 2018-04-24 23:11:28.139).
- [x] Lava Spike is bugged: Lava Spike is only supposed to target players, but XMage allows it to target permanents as well. (reported by iAreNubcake / 2018-04-24 23:12:49.019).
- [x] Walking Ballista Bug: Walking Ballista isn't able to deal damage to a planeswalker at the moment. (reported by Bassiuz / 2018-04-24 23:23:14.964).
- [x] every fucking game: (300 characters max)ya real fucking random fucking junk bullshit (reported by osamamyass / 2018-04-25 00:25:08.941).
- [x] once again: every fucking game (reported by osamamyass / 2018-04-25 00:26:37.1).
- [x] Skyship Stalker + Kwende: Gave skyship stalker first strike with kwende on the battlefield, but he did not gain double strike (reported by Haywire / 2018-04-25 00:42:25.485).
- [x] direct damage to planeswakers: cant redirect damage from a bolt to a planeswalker (reported by joanverdam / 2018-04-25 01:13:51.049).
- [x] I coudnt target plansewalker directly: I was unable to target teferi with wizards lightning (reported by Demaron / 2018-04-25 03:47:14.599).
- [x] Sorcerers Wand: Dominaria Sorcerer's Wand is allowing players to ping creatures when the spell is only planeswalkers or player. Bug. (reported by GODZINGUS / 2018-04-25 05:17:36.051).
- [x] Bolas Player redirect: Bolas walkers need to be able to redirect damage from players to planeswalkers :) (reported by exolith / 2018-04-25 07:01:19.129).
- [x] any target (but not planswalkers): after kicking comet storm the appropriate number of times, i was unable to target planeswalkers for damage (reported by Knighthawke / 2018-04-25 07:09:03.722).

**Comment by LevelX2** [comment]
> osamamyass is so fanny smiley

At least he spends a lot of energy into feedback. :-)

**Comment by theelk801** [comment]
So we're still getting reports of players being unable to properly utilize the damage errata, but things have been fine for me in testing. Either there's something weird going on or some people haven't updated yet.

**Comment by LevelX2** [comment]
> So we're still getting reports of players being unable to properly utilize the damage errata, but things have been fine for me in testing

All that feedback was created before the update to V3 this morning 08:00 CET

**Comment by theelk801** [comment]
Oh good, I was wondering when the server would reset.

**Comment by LevelX2** [comment]
So you can check the feedback messages related to targeting problems, you think that is already fixed.

We will get the feedback from V3 if something not solved yet.

**Comment by Zerrisx** [comment]
Checked every un-checked issue from here.  All of them appear fixed (or duplicates) except:

> dire fleet daredevil + wizard's lightning: i cast wizard's lightning from opponent's graveyard with dire fleet daredevil (no other creatures, no wizards on board) and it cost R, like with cost reduction applied (reported by Rybsson / 2018-04-23 19:11:07.715).

Which I wrote up in #6857.

As such, I think this issue can be closed.

---

❌ Feedback from 1.4.29V1
 
* [x]  Rollback: It seems that whenthe opponent asks for a Rollback, my turn just skips automatically. My opponent has to rollback again in order to go back to normal. (reported by The_Vigil / 2018-04-21 13:28:15.171).

Duplicate of #4309

 * [ ]  view hideaway lands: it would be nice to see what I put under a hideaway land in case it slips my mind (reported by Knighthawke / 2018-04-21 23:13:34.072).

Fixed; if you double click the facedown card, it shows it to you.
![image](https://user-images.githubusercontent.com/59861365/87860237-64b5ae80-c901-11ea-8a91-1c033e94a895.png)

* [ ]  Commander game priority freeze: Whilst playing commander priority seemed to be given to two players at the same time which caused the game to freeze (reported by JoElVago / 2018-04-22 04:24:57.592).

Cannot reproduce, presume fixed.

 * [ ]  soubond bug: soul bond is not triggering when another creature enters the battlefield (reported by Knighthawke / 2018-04-22 05:34:44.993).

Cannot reproduce, presume fixed.

 * [ ]  saga's aren't working as expected: According to the rules, sagas should be on battlefield until after the third chapter resolves. Right now I can't bounce the saga in response to the third chapter because they leave the

Cannot reproduce, presume fixed.

* [ ]  xmage still doesnt work properly with multiplayer games and conceding: xmage still doesnt work properly with multiplayer games and conceding (reported by EvilGeek / 2018-04-20 18:10:02.031).

Duplicate of #3839

❌ Feedback from 1.4.29V2

* [ ]  fix bugs: my opponent tapped eldrazi temple for 2 mana while my damping sphere was in play. this is ANOTHER bug as it doesnt work that way (reported by chaosz41 / 2018-04-22 13:15:01.875).

Cannot reproduce, presume fixed.

* [ ]  Como rankean a los players: i dont understand, when i play a ranking match i lose 6 point, but when i win a ranking match i get a 3 point. (reported by whiteraven / 2018-04-23 04:30:12.773).

This is how ELO systems work; not a bug.

* [ ]  dire fleet daredevil + wizard's lightning: i cast wizard's lightning from opponent's graveyard with dire fleet daredevil (no other creatures, no wizards on board) and it cost R, like with cost reduction applied (reported by Rybsson / 2018-04-23 19:11:07.715).

Totally still a bug!  Wrote up in #6857.

* [ ]  Teshar bug: Teshar doesn't trigger on Sram (reported by casualsmash / 2018-04-24 01:23:48.443).

It won't trigger if there are no valid targets, but otherwise, cannot reproduce.  Presume fixed.

* [ ]  timer issues: In my game, the opponent's clock was ticking and I had no interactions possible, but somehow I lost to idleness (reported by jerjern / 2018-04-24 04:03:55.792).

Hard to track from just this description; not really enough information for a useful bug report.

* [ ]  eldrazy temple: Eldrazy temple still gives 2 mana under damping sphere to eldrazy spells (reported by Dadupe / 2018-04-24 11:53:41.708).

Cannot reproduce, presume fixed.

* [ ]  Humility is bugged: I just finished playing a game where I had Humility on the board against firesong and sunspeaker where he played earthquake and still gained all the life from the earthquake. (reported by schulze961 / 2018-04-25 00:43:24.017).

Cannot reproduce, presume fixed.

* [ ]  Pitiless plunderer and boardwipes: It specifically says in gatherer that if multiple things die at the same time as pitiless plunderer, I get those treasures. However, this doesn't happen currently (reported by Haywire / 2018-04-25 02:38:10.345).

Appears to be fixed.
![image](https://user-images.githubusercontent.com/59861365/87860431-0a1d5200-c903-11ea-85e4-8f2ec3eac493.png)

* [ ]  A scrollbar for your hand cards: When playing with infinite/ all Cards from your libary on your hand temporarely it would help to have a scrollbar to see all the cards (reported by Deathreaver / 2018-04-25 02:39:35.3).

Appears to be fixed.
![image](https://user-images.githubusercontent.com/59861365/87860409-e0fcc180-c902-11ea-95a6-979fe9e35e06.png)

* [ ]  Master of the wild Hunt Bug: When Master of t.w.h activates his ability it asks the target of the ability to select a wolf, but you are not able to it may be because he has hexproof on wolves so you are not able to select them therefore youre unable to play the game (reported by HoodieChris / 2018-04-25 05:30:47.687).

Cannot reproduce, presume fixed.

* [ ]  Asceticism + Master of the Wild Hunt = Broken Game: Using Asceticism with Master of the Wild Hunt's effect caused the game to want the opponent to choose creatures on mine to deal damage to but cannot due the hexproof effect that Asceticism gives. (reported by BigHam / 2018-04-25 05:35:52.177).

Cannot reproduce, presume fixed.

* [ ]  Sorting by color doesn't work.: When I hit the color tab in the Deck Editor nothing happens while it should sort the cards by their one color or multicolored. This has been broken for quite a few patches now. (reported by BigHam / 2018-04-25 05:37:54.896).

Cannot reproduce, presume fixed.

**Comment by JayDi85** [comment]
> Which isn't enough information to write up a meaningful issue.

Can be related to timer issue (see #6481 and #4349);

**Comment by Zerrisx** [comment]
> 
> 
> > Which isn't enough information to write up a meaningful issue.
> 
> Can be related to timer issue (see #6481 and #4349);

It's most likely #4349, but I can't be sure from the information provided.
